tests/pql/test_query.py

```diff
@@ -250,3 +250,16 @@ def test_query_result():
     )
 
     assert result is not None
+
+
+def test_multiply_select_query():
```

Comment:
@Monikaa1909, @patrycjarecko: czemu dotykacie testy, ktore nie dotycza waszego kodu?

Reply:
@wvffle poprawka -> widzę, że to są zmiany z commita Filipa, który był jako ostatni w masterze [4cccc24](https://github.com/wvffle/ATS-project-simple/commit/4cccc24fc2037ef2049c798e0fba0ae1a4686fe2)

więc nie wiem czemu jest w takiej formie tutaj @Monikaa1909